migrator/migrations/m_176_to_m_177_network_baselines_cidr/networkentitystore/postgres_plugin.go

```diff
@@ -0,0 +1,407 @@
+package networkentitystore
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/jackc/pgx/v4"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	ops "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
```

Comment:
Shouldn't use this package in a migration.  frozenschema/v73 is probably what you need.

Reply:
Done